sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JdbcUtils.scala

```diff
@@ -211,6 +211,52 @@ object JdbcUtils extends Logging {
   }
 
   /**
+   * Returns the schema if the table already exists in the JDBC database.
+   */
+  def getSchema(conn: Connection, url: String, table: String): Option[StructType] = {
+    val dialect = JdbcDialects.get(url)
+
+    Try {
+      val statement = conn.prepareStatement(dialect.getSchemaQuery(table))
+      try {
+        getSchema(statement.executeQuery(), dialect)
+      } finally {
+        statement.close()
+      }
+    } match {
+      case Success(v) =>
+        Some(v)
+      case Failure(e) =>
+        None
+    }
```

Comment:
I see. Thank you for review! I wrote that by keeping the same logic in [tableExists](https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JdbcUtils.scala#L66-L80
) because I thought the guideline is about using return `Try` as a return value before this PR. 

Sure, I'll remove the usage of `Try/Success/Failure`.